doc/nrf/protocols/matter/getting_started/tools.rst

```diff
@@ -164,132 +164,126 @@ For more information about the ZAP tool, see the official `ZCL Advanced Platform
 Installing the ZAP tool
 =======================
 
-To install the ZAP tool using the recommended method, complete the following steps:
+You can install the ZAP tool either automatically or manually:
 
-1. Download the ZAP package containing pre-compiled executables and libraries and extract it:
+.. tabs::
 
-   a. Open your installation directory for the |NCS| in a command line.
-   #. Navigate to :file:`modules/lib/matter`.
-   #. Run the helper script to download and extract the ZAP package:
+   .. tab:: Automatic installation
 
-      .. parsed-literal::
-         :class: highlight
+      You can use the Matter West command to automatically download and install the appropriate ZAP tool version dedicated to your current Matter revision.
+      Both ``zap-gui``, and ``zap-generate`` commands automatically install the right version of the ZAP tool in the ``.zap-install`` directory within the |NCS|.
+      After the first installation, if the version of the Matter SDK did not change the installed version will be used for the next command invocations.
+      This tool checks the version of the Matter stack, which is located in the default location based on the ``ZEPHYR_BASE`` variable.
+      If you wish to install the ZAP tool for a different Matter version, provide the path to the desired Matter SDK location using the ``-m`` or ``--matter-path`` command argument.
 
-         python scripts/setup/nrfconnect/get_zap.py -l *zap_location* -o
+      To install the ZAP tool according to the default Matter SDK location, use the following command:
 
-      In this command:
+      .. code-block:: console
 
-      *  *zap_location* corresponds to the directory where the ZAP tool package is to be extracted.
-      *  The `-o` argument in the command is used to allow overwriting files, if they already exist in the given location.
-         Otherwise the script will display prompt during download and ask for user consent to overwrite the files directly.
+         west zap-gui
 
-#. Verify the ZAP tool version by comparing the script output with the following possible cases:
+      To install the ZAP tool for a specific Matter version, use the following command and specify the *path_to_Matter_SDK*:
 
-   * Case 1: If your currently installed ZAP version matches the recommended one, you will see a message similar to the following one:
+      .. code-block:: console
 
-     .. code-block::
+         west zap-gui -m <path_to_Matter_SDK>
+      
+      .. note::
 
-        Your currently installed ZAP tool version: 2022.12.20 matches the recommended one.
+         This feature is relevant using Matter in |NCS| rather than the Matter upstream repository, so providing a path to the incompatible version of the Matter SDK may cause unpredictable errors.
```

Comment:
```suggestion
         Providing a path to an incompatible version of the Matter SDK may cause unpredictable errors.
```
I think this should be enough, we don't necessarily need to specify why the user would need to install a specific version.